chart/values.yaml

```diff
@@ -50,7 +50,7 @@ timescaledb-single:
 #   https://github.com/timescale/promscale/tree/master/helm-chart
 promscale:
   enabled: true
-  image: timescale/promscale:0.11.0
```

Comment:
This will be included in https://github.com/timescale/tobs/pull/438 so let's hold with merging this PR.